config/example.ini

```diff
@@ -132,7 +132,7 @@ kinto.history.exclude_resources = /buckets/preview
 #
 # Kinto changes
 #
-kinto.changes.resources = /buckets/destination/collections/destination
+kinto.changes.resources = /buckets/blog/collections/articles
```

Comment:
On our servers (DEV and STAGE), this would be `/buckets/main-workspace/collections/product-integrity`